app/services/nebraska_weekly_hours_attended_calculator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Service to calculate full days used in Nebraska by specific kids
+class NebraskaWeeklyHoursAttendedCalculator
```

Comment:
@JonErvin here's the most important part if you want to take a look at this PR! :)